tp/containers/k8s-kind-install.fr.md

```diff
@@ -0,0 +1,49 @@
+# Travaux Pratiques: Kubernetes - KIND
+
+## Introduction
+
+Dans ce TP, nous allons déployer un cluster avec KIND (Kubernetes in Docker).
```

Comment:
Maybe a link to the kind website (https://kind.sigs.k8s.io/) could fit in the introduction ?